### PR TITLE
Clarify ranges of normalized values for unorm, snorm packing/unpacking

### PR DESCRIPTION
**Pull request by dneto0**
Fixes: #2903

### CODE REVIEW

File: wgsl/index.bs [hunk 1]
```diff
@@ -14352,6 +14356,13 @@ do not correspond directly to types in WGSL.
 This enables a program to write many densely packed values to memory, which can
 reduce a shader's memory bandwidth demand.
 
+Each builtin applies the *inverse* of a [=channel transfer function=] to several input values, then combines
+their results into a single output value. Teh resul
```
Comment (alan-baker) [inline wgsl/index.bs:14360]: Typo
```suggestion
their results into a single output value. The result
```

File: wgsl/index.bs [hunk 2]
```diff
@@ -14352,6 +14356,13 @@ do not correspond directly to types in WGSL.
...
+
+Note: For packing unorm values, the normalized floating point values are in the interval [0,1].
+
+Note: For packing snorm values, the normalized floating point values are in the interval [-1,1].
```
Comment (alan-baker) [inline wgsl/index.bs:14364]: For consistency with the previous text, I'd suggest:
```suggestion
Note: For packing unorm values, the normalized floating point values are in the interval [0.0, 1.0].

Note: For packing snorm values, the normalized floating point values are in the interval [-1.0, 1.0].
```

File: wgsl/index.bs [hunk 3]
```diff
@@ -14445,6 +14456,12 @@ data formats that do not correspond directly to types in WGSL.
 This enables a program to read many densely packed values from memory, which can
 reduce a shader's memory bandwidth demand.
 
+Each builtin breaks up an input value into channels, then applies a [=channel transfer function=] to each.
+
+Note: For unpacking unorm values, the normalized floating point result is in the interval [0,1].
+
+Note: For unpacking snorm values, the normalized floating point result is in the interval [-1,1].
```
Comment (alan-baker) [inline wgsl/index.bs:14463]: Similarly:
```suggestion
Note: For unpacking unorm values, the normalized floating point result is in the interval [0.0, 1.0].

Note: For unpacking snorm values, the normalized floating point result is in the interval [-1.0, 1.0].
```

File: wgsl/index.bs [hunk 4]
```diff
@@ -3494,6 +3494,10 @@ The channel formats used to define those texel formats are listed in the
 The last column specifies the conversion from the stored channel bits to the value used in the shader.
 This is also known as the <dfn noexport>channel transfer function</dfn>, or CTF.
 
+Note: The channel transfer function for 8unorm maps {0,...,255} to the floating point interval [0.0, 1.0].
+
+Note: The channel transfer function for 8snorm maps {-128,...,127} to the floating point interval [-1.0, 1.0].
```
Comment (alan-baker) [inline wgsl/index.bs:3499]: Take either this suggestion or the next two about the ranges for consistency.
```suggestion
Note: The channel transfer function for 8unorm maps {0,...,255} to the floating point interval [0,1].

Note: The channel transfer function for 8snorm maps {-128,...,127} to the floating point interval [-1,1].
```

**Review by alan-baker — CHANGES_REQUESTED** [review]
Pick either of the two consistency sets of suggestions.

File: wgsl/index.bs (hunk 1 above)
Comment (dj2) [reply wgsl/index.bs:14360]: The result seems like a dangling sentence?

Comment (dneto0) [reply wgsl/index.bs]: sigh.  Yep. Removed.

File: wgsl/index.bs (hunk 2 above)
Comment (dneto0) [reply wgsl/index.bs]: took this one.

File: wgsl/index.bs (hunk 4 above)
Comment (dneto0) [reply wgsl/index.bs:3499]: took the float point one.

**Review by alan-baker — APPROVED** [review]

**Pull request merged**